kv/dist_sender.go

```diff
@@ -252,6 +252,11 @@ func (ds *DistSender) verifyPermissions(args proto.Request) error {
 		})
 }
 
+// lookupOptions capture additional options to pass to InternalRangeLookup.
+type lookupOptions struct {
```

Comment:
I don't have anything in mind, but threading a random bool through the range descriptor stuff was a heinous crime, so I opted for this.